benchmarks/src/main/java/org/apache/druid/benchmark/query/ScanBenchmark.java

```diff
@@ -100,6 +100,7 @@
 @Fork(value = 1)
 @Warmup(iterations = 10)
 @Measurement(iterations = 25)
+@SuppressWarnings("SSBasedInspection") // static field(s) cannot be final because variable is not initilized.
```

Comment:
If this is about `ordering` field, I think it could be an instance field. The order of static and instance field is also disturbed.